src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(victim) {
+    const findAnimal = Animal.alive.find(name => name === victim);
```

Comment:
the **argument** in method `bite` is **object**. You don't have to find the object because it's already given. 

![image](https://user-images.githubusercontent.com/108505176/186929810-448e5269-a354-4836-9e91-a4e674df3dd3.png)